applications/FluidDynamicsApplication/python_scripts/fluid_solver.py

```diff
@@ -95,8 +96,8 @@ def PrepareModelPart(self):
             self._ReplaceElementsAndConditions()
             ## Executes the check and prepare model process
             self._ExecuteCheckAndPrepare()
-            ## Set buffer size
-            self.main_model_part.SetBufferSize(self.min_buffer_size)
+            ## Set and fill buffer
+            self._SetAndFillBuffer()
```

Comment:
I have a doubt. The initial condition should be impsed after setting the buffer size and before filling the buffer. It is not relevant if the buffer size is 2, but with higher values there will be uninitialized steps (I have a scheme with buffer size 4)
Am I wrong?

Reply:
mmm... I think I don't get your point at all. Either if you impose your initial condition by using a custom process that fills the entire buffer or by customizing the `ModifyAfterSolverInitialize` in your `MainKratos.py` the `_SetAndFillBuffer` would have been already called.